src/app/pages/FrontPage/index.test.jsx

```diff
@@ -188,34 +197,114 @@ describe('Front Page', () => {
         expect(section.getAttribute('role')).toEqual('region');
       });
     });
+  });
+
+  describe('Ads', () => {
```

Comment:
```suggestion
  describe('Front Page with Ads', () => {
```

Reply:
never mind this, I just noticed the outer describe block is `Front Page`

Reply:
It should be fine as it is, since this `decribe` block is under the `Front Page` one:
<img width="665" alt="Screenshot 2020-07-16 at 23 00 55" src="https://user-images.githubusercontent.com/46446236/87805136-c9630180-c84c-11ea-983a-f4602288d24a.png">